npm/builder.js

```diff
@@ -27,10 +27,16 @@ function Builder () {
     'node_modules/opal-runtime/src/opal.js',
     'build/asciidoctor-lib.js'
   ];
-  if (process.env.ASCIIDOCTOR_CORE_VERSION) {
-    this.asciidoctorCoreVersion = process.env.ASCIIDOCTOR_CORE_VERSION;
+  var asciidoctorRef = process.env.ASCIIDOCTOR_CORE_VERSION;
```

Comment:
I think we should add an example in a comment to mention that `ASCIIDOCTOR_CORE_VERSION` can be a reference to a fork (user/project#branch) or a tag (v1.5.6.1).